HolyshiftService/HolyShiftDbContext.cs

```diff
@@ -12,6 +12,7 @@ public HolyShiftDbContext(DbContextOptions<HolyShiftDbContext> options, IOptions
         _config = config.Value;
     }
 
+    public DbSet<HolyShiftUser> Users { get; set; } = null!;
```

Comment:
I prefer `= default!` and let the compiler decide that `null` is the one suitable here.